compiler/src/iree/compiler/Codegen/Utils/CPUUtils.cpp

```diff
@@ -63,4 +64,25 @@ FailureOr<Operation *> getRootOperation(ArrayRef<Operation *> computeOps) {
   return rootOperation;
 }
 
+StringAttr getPeelAttrName(MLIRContext *ctx) {
+  return StringAttr::get(ctx, "enable_loop_peeling");
```

Comment:
Nit: Define a static char array with the name an use that.
```
static const char[] kLoopPeeling = "enable_loop_peeling";
```
and use `kLoopPeeling` here.